main.py

```diff
@@ -0,0 +1,49 @@
+from flask import Flask, render_template, request, redirect
+from flask_mysqldb import MySQL
+ 
+app = Flask(__name__)
+
+app.config['MYSQL_HOST'] = 'localhost'
+app.config['MYSQL_USER'] = 'root'
+app.config['MYSQL_PASSWORD'] = 'Shikhar@970'
+app.config['MYSQL_DB'] = 'mysqldb'
```

Comment:
These are Database connection secrets. Ideally, we should not expose them on codebase. Can you create a card to remove these secrets?

Reply:
Yeah! sure Subbu. I will create a Trello card for that name as _Database Secrets_, Is it good Subbu?

Reply:
`Load database secrets from environment variables` would be good

Reply:
That's the good name  for the story